### PR TITLE
💎(site) Video Cards

### LINKED ISSUE #263: Module: Video Cards
Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A4714

### PR DESCRIPTION
**Pull request by nickalcantara**
Introduces a Video card component.

- Resolves #263

### CODE REVIEW

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks for your work here! A couple comments, but the most important one is the use of the actual YouTube video thumbnails instead of Drive images for the videos.

Thanks!

File: site/en/_data/home.yml [hunk 1]
```diff
@@ -106,6 +106,30 @@ community:
...
     heading: h2
+videos:
+  title: Watch the latest videos
+  items:
+    - video:
+        title: Developer Keynote at Google I/O '21
+        url: https://www.youtube.com/watch?v=FPuaaYpUd5s
+      capture:
+        src: https://drive.google.com/uc?id=13gLkQxETrT-U2JOAdLHL7MD3Hrq1r7sO
+        alt: 'Google IO: Input matters for Chrome OS'
```
[the remark this answers is not in the document]
Comment (nickalcantara) [reply site/en/_data/home.yml:117]: Whoops! While I meant to swap these out for assets that were previously handed off, I didn't know we could do that with YouTube thumbnails! very convenient.

File: site/sass/components/_video-card.scss [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+    position: relative;
 
-    > * {
-      grid-area: 1 / -1;
+    // Use the same aspect ratio as YouTube does for thumbnails to minimize letterboxing.
+    &::before {
+      content: '';
+      display: block;
+      padding-top: 56.25%;
+      width: 100%;
```
Comment (Snugug) [inline site/sass/components/_video-card.scss:15]: Can we use `aspect-ratio` here if it's supported? You can use the same TODO that I've got [here](https://github.com/chromeos/chromeos.dev/blob/e2808f28c73473d689623bc93f70bc2d97b6d8d6/site/sass/components/_embed-responsive.scss), or use that class directly if you'd like, for iOS < 15

**Review by Snugug — CHANGES_REQUESTED** [review]
There's one small change here, but otherwise this looks GTG! Also, it looks like there's a merge conflict that should be quick to resolve

**Comment by Snugug** [comment]
@nickalcantara There appear to be merge conflicts here; if you can resolve them I can take another look. Thanks!

**Comment by nickalcantara** [comment]
@Snugug conflicts resolved!

File: site/sass/components/_video-card.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    overflow: hidden;
+    position: relative;
+
+    // Use the same aspect ratio as YouTube does for thumbnails to minimize letterboxing.
+    &::before {
+      content: '';
+      display: block;
+      padding-top: 56.25%;
+      width: 100%;
+    }
```
Comment (Snugug) [inline site/sass/components/_video-card.scss:16]: Looks like this comment got missed in the merge conflicts.

Can we use `aspect-ratio` here if it's supported? You can use the same TODO that I've got [here](https://github.com/chromeos/chromeos.dev/blob/e2808f28c73473d689623bc93f70bc2d97b6d8d6/site/sass/components/_embed-responsive.scss), or use that class directly if you'd like, for iOS < 15

**Review by Snugug — CHANGES_REQUESTED** [review]
Looks like my previous comment got missed in the midst of fixing merge conflicts. Can we use `aspect-ratio` instead of the padding hack?

Thanks!

**Comment by nickalcantara** [comment]
> Looks like my previous comment got missed in the midst of fixing merge conflicts. Can we use `aspect-ratio` instead of the padding hack?
> 
> Thanks!

Whoops - updated!

**Review by Snugug — CHANGES_REQUESTED** [review]
A small rewrite of the `aspect-ratio` that I'm going to accept then merge so we can close this out.

**Review by Snugug — DISMISSED** [review]

**Review by Snugug — APPROVED** [review]

**Pull request merged**